README.md

```diff
@@ -98,7 +101,31 @@ docker compose up -d postgres
 uv run pytest
 ```
 
-Note: the test_mais.py file depends on the MaIS API being configured with production credentials.  If no credential are available in the environment variables, the tests will be skipped.  If UAT credentials are supplied, some of the tests may fail, since they assert checks against production data.
+### Test Setup
+
+In order for some of the tests to run, they will need to hit actual APIs.  In order to do this,
+they will need to be properly configured with keys and URLs.  These need to be placed in the .env file.
+
+For Google drive tests, update your .env with values shown below / pulled from vault as indicated:
+
+```
+AIRFLOW_VAR_GOOGLE_CONNECTION="google_cloud_default"
+AIRFLOW_VAR_GOOGLE_SERVICE_ACCOUNT_JSON=${get from vault at `rialto-airflow/prod/google_service_account_json`}
```

Comment:
might be nice to have the full vault path (need the start of the path, `puppet/application`)